infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,88 @@
+  4. [High available PostgreSQL 9.6+](https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/identity-provider#postgresql-operator) (Harbor does not handle the deployment of HA of database)
+  5. [High available Redis](https://github.com/bitnami/charts/tree/master/bitnami/redis#redis-chart-packaged-by-bitnami) (Harbor does not handle the deployment of HA of Redis)
+  6. PVC that can be shared across nodes (i.e., with `ReadWriteMany` access mode) or external object storage
 
-  2. Annotations concerning the configuration of the Ingress controller.
-```yaml
-ingress:
-  enabled: true
-  path: /
-  hosts:
-    - <registry.domain.name>
-  annotations:
-    cert-manager.io/cluster-issuer: letsencrypt-production
-  labels: {}
-  tls:
-    - hosts:
-      - <registry.domain.name>
+### Redis Configuration
+For the installation of Redis service, please see the link on the pre-requisites section about the `High available Redis`.
+In our architecture we have a [Redis-Sentinel](https://redis.io/topics/sentinel) service, instead of [Redis Cluster](https://redis.io/topics/cluster-tutorial), because with this architecture Sentinel manages automatically the failover of Master.
+It is possible have a `Redis-Sentinel ` architecture by setting the following parameter in the file values of redis_service_values.yaml file:
+  ```yaml 
+  sentinel.enabled=true
+  ```
+In the file you have to set an IP for the ClusterIP of the sentinel where after harbor is attached in the cluster and a name for your master. If you want have the replicas of redis running in different nodes you have to set the replica.podAntiAffinityPreset parameter to soft or hard, depending what you need. Please see this [link](https://docs.bitnami.com/tutorials/assign-pod-nodes-helm-affinity-rules/) to see the differences.
+To deploy Redis service, please see the link on the pre-requisites section or it is possible to proceed as follow:
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/redis
+$ helm upgrade --install redis bitnami/redis -f values.yaml
 ```
-
-  3. Configuration of the persistent disk. Please dimension this disk appropriately, as VM images may be rather large and hence you may fill up this space in a very short time.
-```yaml
-persistence:
-  accessMode: 'ReadWriteOnce'
-  enabled: true
-  size: 500Gi
-  storageClass: 'rook-ceph-block'
+the last commannd install/upgrade the chart with the release name `redis` and apply the configuration specified by the `values.yaml`.
+  
+### Postgres Configuration
+For the installation of Postgres service, please see the link on the pre-requisites section about the `High available PostgreSQL 9.6+` or it is possible to proceed as follow:
+```bash
+$ kubectl apply -f values.yaml
 ```
+This command create the database and apply the confguration specified by the `values.yaml`.
```

Comment:
Please, make this section more explicative. With no prior knowledge, would you be able to create the database from the instructions written here? I doubt so, as the `values.yaml` file does not even exist. Additionally, you should mention a prerequisite is the postgres operator, explain why and add a link to the documentation to install it (within the CrownLabs repo).